kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/KVDoc.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 Responsive Computing, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.responsive.kafka.internal.db.mongo;
+
+import org.bson.types.ObjectId;
+
+public class KVDoc {
+
+  ObjectId _id;
+  byte[] key;
+  byte[] value;
+  long epoch;
+
+  public KVDoc() {
+  }
+
+  public KVDoc(final byte[] key, final byte[] value, final long epoch) {
+    this.key = key;
+    this.value = value;
+    this.epoch = epoch;
+  }
+
+  public ObjectId _id() {
+    return _id;
+  }
+
+  public void set_id(final ObjectId _id) {
```

Comment:
What is this, C++? 😛 
```suggestion
  public void setId(final ObjectId id) {
```
ditto everywhere else

Reply:
ah I have actually updated this and forgot to push the update, but just so you know the original reason is that Mongo has a field that is autogenerated called `_id` and in order to access it I thought the name here has to match. but they actually will magically replace any `id` with `_id` underneath the hood so it isn't necessary.